conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -936,6 +936,13 @@ msgid "Unable to send email activation link. Please try again later."
 msgstr ""
 
 #: common/djangoapps/third_party_auth/models.py
+msgid ""
+"If this option is not selected, users will not be presented with the provider "
+"as an option to authenticate with on the login screen, but manual "
+"authentication using the correct link is still possible."
+msgstr ""
+
+#: common/djangoapps/third_party_auth/models.py
```

Comment:
@bdero @haikuginger As far as I know, we should not be including these `LC_MESSAGES` changes. It makes the diff messy, is prone to conflicts, and isn't necessary. I believe someone at edX updates them on a regular basis for the whole platform repo.